src/views/LoginView.vue

```diff
@@ -1,66 +1,133 @@
 <template>
   <div>
     <main>
-      <img id="login-image" src="../assets/img/bier.png" alt="logo" />
+      <img
+          id="login-image"
+          src="@/assets/img/bier.png"
+          alt="logo"
+      />
       <h1>{{ $t('login.SudoSOS Login') }}</h1>
-      <Button id="login-gewis-button" @click="loginViaGEWIS" severity="success"
-        ><img id="gewis-branding" src="../assets/img/gewis-branding.svg" alt="GEWIS" />{{
-          $t('login.Login via GEWIS')
-        }}</Button
+      <Button
+          id="login-gewis-button"
+          @click="loginViaGEWIS"
+          severity="success"
       >
-      <hr />
-      <form id="login-form" @submit.prevent="ldapLogin">
-        <!--      TODO: Form validation with vee-validate -->
-        <label for="username">{{ $t('login.Username') }}</label>
-        <InputText
-          id="username"
-          type="text"
-          v-bind="username"
-          :placeholder="$t('login.Enter username')"
-        />
-        <label for="password">{{ $t('login.Password') }}</label>
-        <InputText
-          id="password"
-          type="password"
-          v-bind="password"
-          :placeholder="$t('login.Enter password')"
+        <img
+            id="gewis-branding"
+            src="@/assets/img/gewis-branding.svg"
+            alt="GEWIS"
         />
-        <Button type="submit" id="login-button" severity="danger">{{ $t('login.Login') }}</Button>
-        <div class="password-reset" @click="resetPassword">{{ $t('login.Password reset') }}</div>
-      </form>
+        {{ $t('login.Login via GEWIS') }}
+      </Button>
+      <hr />
+      <Form id="login-form" @submit="loginHandler">
+        <label
+            id="input-description"
+            for="username"
+        >
+          {{ $t('login.Username') }}
+        </label>
+        <span class="p-float-label with-error">
```

Comment:
The error message right now moves the whole UI, I'd suggest making it so that it fills a whitespace that is already there.